pro/src/new_components/BoxFormLayout/BoxFormLayoutFormHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import cn from 'classnames'
+import React from 'react'
+
+import style from './BoxFormLayout.module.scss'
+
+interface BoxFormLayoutFormHeader {
+  className?: string
+  text_primary: string
+  text_secondary: string
+}
+
+const FormHeader = ({
+  className,
+  text_primary,
+  text_secondary,
+}: BoxFormLayoutFormHeader): JSX.Element => (
+  <div className={cn(style['box-form-layout-form-header'], className)}>
+    <span className={style['box-form-layout-form-header-secondary']}>
+      {text_secondary}
+    </span>{' '}
+    :{' '}
+    <span className={style['box-form-layout-form-header-primary']}>
```

Comment:
```suggestion
    </span>
    {' : '}
    <span>
```
je trouve ça plus joli comme ça :)
+ je crois que tu n'utilise pas la classe